### DISCUSSION TITLE
Dashboard in github gets truncated for mono repo

### DISCUSSION THREAD

**Discussion by ns-ggeorgiev** [renovatebot/renovate#25506] [Suggest an Idea]
### Tell us more.

I am having a mono repo that has big number of dependencies. For a while I was considering with proper filtration and addressing issues from the top we can manage. We have done this for long time and the problems just grow, since more teams adopting the tool and create regex managers etc.

The long story short, the size limited truncated issue based dashboard does not work well for us. I am wondering if the renovate team would like to consider other options to deal with this problem.

1. Multi page dashboard. Obviously not great but instead of truncating the renovate should create/update and extra issue where it can add the rest of the dashboard information.

2. Target page dashboard. Similar to the first one, but instead of breaking the report in pages, allowing for dashboard name. This way the use can split different dependencies to go to different dashboard. This could be used to create team specific dashboards for example... and obviously could be utilized to break down the dashboards until they fit in the size limit.

3. Renovate dashboard web service. What if renovate team offer a web service to which the dashboard data can be uploaded. This will allow for way more user friendly functionality, including multi-repo dashboard, and most importantly unlimited in size. That also could make for useful API queries.

**Comment by rarkins** [comment]
For the hosted Renovate App, I think the full dashboard content is available at developer.mend.io

Back to the repo, I would prefer to split the dashboard into issue + comments so that it's all in once place. We've considered supporting multiple dashboards before, but I suspect it could rapidly get quite complex (e.g. if one branch was configured for two different dashboard names)

**Reply by ns-ggeorgiev** [reply]
is developer.mend.io an option for self hosted renovate and is there self hosting option for the app?

**Reply by rarkins** [reply]
No/No
